File: contest3/eszett.py
#o meu refrigerador nao funciona
s = input()

tam = len(s)

i_eszett = -1;
qnt_eszett = 0;

i = 0
while i < tam:
    
    if s[i] == 'S' and s[i+1] == 'S':
            
        i_eszett = i;
        qnt_eszett+=1;
        
        if i + 2 < tam - 1 and s[i+2] == 'S':
            
            qnt_eszett+=1;
            i += 3;
            break;
            
        else:
            
            i+= 2;
            
            
    
    i += 1
    
print(s.lower())
s = s.lower()

if i_eszett != -1:
        
        if qnt_eszett == 1:
           
            primeira_parte = s[0:(i_eszett-1)]
            print(primeira_parte)
            #print(s[0:(i_eszett-1)] ++ 'B' ++ s[i_eszett + (2:tam-1]))
            
            
            
        

File: contest3/eszett.cpp
//link

#include <bits/stdc++.h>
#define ll long long int
using namespace std;

int main(){
    
    ios_base::sync_with_stdio(false);
    cin.tie(0);

    
    string s; cin >> s;
    int tam = s.size();
    int i_eszett = -1;
    int qnt_eszett = 0;
    
    for (int i = 0; i < tam - 1; i++){

        if (s[i] == 'S' and s[i+1] == 'S'){
            
            i_eszett = i;
            qnt_eszett++;
            
            if(i + 2 < tam - 1 and s[i+2] == 'S'){
                
                qnt_eszett++;
                i += 3;
                break;
                
            }
            else{
                
                i+= 2;
                
                
            }
            
            
        }
    }
    
    for (int i = 0; i < tam; i++){

        //cout << tolower(s[i]);
        //cout << s[i] +32;
        
    }
    
    cout << '\n';
    
    
    if(i_eszett != -1){
        
        if (qnt_eszett == 1){
            
            for(int i = 0; i < i_eszett; i++){
                
                cout << s[i];
                
                
            }
            
            cout << 'B';
            
            for(int i = i_eszett + 2; i < tam; i++){
                
                cout << s[i];
                
                
            }
            
            cout << '\n';
            
        }
        
        
    }
    

    return 0;
}
